matplotlib_pyodide/html5_canvas_backend.py

```diff
@@ -428,8 +428,8 @@ def __init__(self, canvas, num):
         self.set_window_title("Figure %d" % num)
         self.toolbar = NavigationToolbar2HTMLCanvas(canvas)
 
-    def show(self):
-        self.canvas.show()
+    def show(self, *args, **kwargs):
+        self.canvas.show(*args, **kwargs)
```

Comment:
Indeed it does look like some canvas-related methods have a `show` method with no arguments, and others accept the `block` argument, so this would probably be the way to go.